### PR TITLE
Edit Checking

### PR DESCRIPTION
**Pull request by j-jasik**
Added some keyup functions to check that from_path and to_url are valid in the edit fields.

### CODE REVIEW

File: tinker/templates/admin/redirects/home.html [hunk 1]
```diff
@@ -164,12 +164,35 @@
                     input.val("/" + value); // add in a slash
                     value = input.val();
                 }
+                if (value[0] == '/' && value[1] =='/'){
+                    input.val(input.val().replace('//','/')); // eliminates a lead double slash
```
Comment (c-schwarze) [inline tinker/templates/admin/redirects/home.html]: technically, this still allows a triple slash to be changed to a double slash - which won't be changed to a single slash

Comment (c-schwarze) [reply tinker/templates/admin/redirects/home.html]: Maybe do a replace all? And maybe do a string check to see if // is in the string? Or, you could consider just running it everytime regardless.

File: tinker/templates/admin/redirects/home.html [hunk 2]
```diff
@@ -164,12 +164,35 @@
...
+                }
                 delay(function () {
                     loadTableForSearch(value, $('#new-redirect-to').val());
                     $('#spinner').hide();
                 }, 300);
             });
 
+            $('#modal-from-path-edit').keyup(function(){
```
Comment (c-schwarze) [inline tinker/templates/admin/redirects/home.html]: didn't we already have keyup functions? Or am i remembering wrong...

File: tinker/templates/admin/redirects/home.html [hunk 3]
```diff
@@ -164,12 +164,35 @@
...
+                var validRedirect = /(\/)[^\/\s][^\s]*/;
```
Comment (c-schwarze) [inline tinker/templates/admin/redirects/home.html]: this definitely needs a comment. Also, this isn't used?

File: tinker/templates/admin/redirects/home.html (hunk 2 above)
Comment (j-jasik) [reply tinker/templates/admin/redirects/home.html]: We didn't... I think I meant to put them in but I must have gotten distracted by something else and forgot. Better late than never??

File: tinker/templates/admin/redirects/home.html (hunk 3 above)
Comment (j-jasik) [reply tinker/templates/admin/redirects/home.html]: Oh yeah this was my first attempt at doing regular expression checking like Phil suggested. It didn't work, so I'll just delete that line.

File: tinker/templates/admin/redirects/home.html (hunk 1 above)
Comment (j-jasik) [reply tinker/templates/admin/redirects/home.html]: It now checks for more than two slashes and gets rid of all of them

**Pull request merged**